### PR TITLE
Remove an extra newline from the Gemfile

### PR DESCRIPTION
**Pull request by PiotrWald**

### CODE REVIEW

File: lib/happy_gemfile.rb [hunk 1]
```diff
@@ -82,9 +82,9 @@ def self.organize_groups lines=nil
       organized << "\n"
 
       (groups.keys - [:general, :not_gems]).each do |group|
-        organized << "group #{group_line(group)} do" << "\n"
+        organized << "\n" << "group #{group_line(group)} do" << "\n"
```
Comment (pienkowb) [inline lib/happy_gemfile.rb:81]: Wouldn't that create a redundant empty line before the first group?

Comment (PiotrWald) [reply lib/happy_gemfile.rb:81]: Correct. It is fixed now

Comment (pienkowb) [reply lib/happy_gemfile.rb:81]: I invoked `happy_gemfile all` on the Gemfile in this repository and there are two empty lines at the end of the file. I suppose we need to rethink how to add the newline characters.

**Review by pienkowb — APPROVED** [review]
🚀

**Pull request merged**